lib/protocol.ml

```diff
@@ -36,8 +36,8 @@ module Client : CLIENT = struct
     Random.self_init ();
     Random.int (1 lsl 16)
 
-  let marshal ?alloc q =
-    [q.Packet.id, Packet.marshal (Buf.create ?alloc 4096) q]
+  let marshal q =
+    [q.Packet.id, Packet.marshal (Cstruct.create 4096) q]
```

Comment:
What is `?alloc` removed here? Is it no longer necessary for page-aligned allocations in Mirage on Xen?